migrations/20170906142654_place.js

```diff
@@ -0,0 +1,12 @@
+
+exports.up = function (knex) {
+  return knex.schema.createTable('place', (t) => {
+    t.increments().index();
+    t.float('latitude');
```

Comment:
Can you check to see if floats store all the digits of a longitude and latitude? I think in a previous project I tried to do this but the precision wasn't high enough. You might have to use a 'decimal' field and specify how much precision is needed based on the value that the Google map api uses.